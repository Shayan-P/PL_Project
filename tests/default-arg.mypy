def xxx(x=2, y=[1, 2, 3], z=12):
    print(x);
    print(y);
    print(z);
    print(2222);
    ;

xxx(1, 2, 3);
xxx();
xxx(1, 5);
