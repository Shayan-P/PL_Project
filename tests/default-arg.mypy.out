(num-val 1)
(num-val 2)
(num-val 3)
(num-val 2222)
(num-val 2)
(list-val (list (num-val 1) (num-val 2) (num-val 3)))
(num-val 12)
(num-val 2222)
(num-val 1)
(num-val 5)
(num-val 12)
(num-val 2222)
'finished
